css/style.css

```diff
@@ -0,0 +1,462 @@
+  cursor: pointer;
+  height: 340px;
+  width: 220px;
+  position: relative;
+  transform-style: preserve-3d;
+  transition: transform .8s;
+}
+
+.card-wrap__hover:hover {
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
+  transform: rotate(-3.08deg) translateY(-15px);
+  transition: .5s ease;
+}
+
+.card__click {
+  transform: rotateY(180deg);
+}
+
+.card__back {
+  background: url(../img/card.png) no-repeat;
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  -webkit-backface-visibility: hidden;
+}
+
+.card__game-over {
+  background: url(../img/game_over.png) no-repeat;
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  transform: rotateY(180deg);
+  -webkit-backface-visibility: hidden;
+}
+
+.card__winner {
+  background: url(../img/bug_card.png) no-repeat;
+  backface-visibility: hidden;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  transform: rotateY(180deg);
+  -webkit-backface-visibility: hidden;
+}
+
+
+.three-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  height: 340px;
+  width: 770px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+.six-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 750px;
+  width: 770px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+.ten-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 750px;
+  width: 1180px;
+  margin: auto;
+  perspective: 1000px;
+}
+
+@media (min-width: 1025px) and (max-width: 1224px) {
+
+  .menu { margin: 0 0 0 70px; }
+
+  .level_btn {
+    font-size: 40px;
+    line-height: 50px;
+  }
+
+  .ten-card .card-wrap,
+  .ten-card .card__back,
+  .ten-card .card__game-over,
+  .ten-card .card__winner {
+    background-size: 175px 270px;
+    height: 270px;
+    width: 175px;
+  }
+
+  .ten-card {
+    height: 570px;
+    width: 935px;
+  }
+
+}
+
+@media (min-width: 987px) and (max-width: 1024px) {
+  
+  .menu { margin: 0 0 0 70px; }
+
+  .level_btn {
+    font-size: 40px;
+    line-height: 50px;
+  }
+
+  .start-button {
+    font-size: 30px;
+    line-height: 40px;
+    height: 70px;
+    width: 290px;
+  }
+
+  .pictures { display: none; }
+
+  .card-wrap,
+  .card__back,
+  .card__game-over,
+  .card__winner {
+    background-size: 175px 270px;
+    height: 270px;
+    width: 175px;
+  }
+
+  .three-card {
+    height: 270px;
+    width: 575px;
+  }
+
+  .six-card {
+    height: 580px;
+    width: 575px;
+  }
+
+  .ten-card {
+    height: 570px;
+    width: 935px;
+  }
+
+}
+
+@media (min-width: 769px) and (max-width: 986px) {
+
+  .menu { margin: 0 0 0 70px; }
+
+  .menu__select-level { margin-bottom: 70px; }
+
+  .level_btn {
+    margin: 0 0 10px 0;
+    font-size: 30px;
+    line-height: 40px;
+  }
+
+  .level_active::before {
+    top: 15px;
+    width: 15px;
+    height: 15px;
+  }  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5839198)